contracts/SharesERC20.sol

```diff
@@ -1,55 +1,24 @@
 pragma solidity 0.8.13;
 //SPDX-License-Identifier: MIT
 
-import "@openzeppelin/contracts/access/Ownable.sol"; //https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol
```

Comment:
just curious - is this ok to remove? i figured the shares would be `Ownable` potentially?

Reply:
yes should be ok, there is logic for baal only on the admin functions and we do not need any other ownership. I guess it might be worth thinking about transferring to another baal or ejecting the token completely. may be worth some future discussion